lib/game.rb

```diff
@@ -6,27 +6,50 @@
 require_relative "./game_checker"
 
 class Game
-  attr_reader :display, :board, :player_1
-  def initialize(display, presenter, board, players)
-    @display = display
-    @presenter = presenter
-    @board = board
-    @player_1 = players[0]
-    @player_2 = players[1]
+  attr_reader :display, :board, :player_1, :player_2, :active_player
+  def initialize(config_object)
+    @display = config_object[:display]
+    @presenter = config_object[:presenter]
+    @board = config_object[:board]
+    @player_1 = config_object[:player_1]
+    @player_2 = config_object[:player_2]
+    @game_checker = config_object[:game_checker]
 
     @active_player = @player_1
+    @game_over = false
   end
```

Comment:
Yeah that's true. I had to restructure the `Game.play` loop to make this work so that the loop would still run through once before ending during the tests.